playbooks/common/openshift-cluster/redeploy-certificates/registry.yml

```diff
@@ -73,6 +73,9 @@
         --hostnames="{{ docker_registry_service_ip.results.clusterip }},docker-registry.default.svc.cluster.local,{{ docker_registry_route_hostname }}"
         --cert={{ openshift.common.config_base }}/master/registry.crt
         --key={{ openshift.common.config_base }}/master/registry.key
+        {% if openshift.common.version_gte_3_5_or_1_5 | bool %}
+        --expire-days={{ openshift_registry_cert_expire_days | default(730) }}
```

Comment:
I think we issue 1 year certs **now**, you're suggesting a 2 year default expiry here. Do we have it defined somewhere how long we want certs to be valid for?